libraries/AP_Scheduler/AP_Scheduler.h

```diff
@@ -72,6 +72,7 @@ class AP_Scheduler
         const char *name;
         float rate_hz;
         uint16_t max_time_micros;
+        uint8_t priority;
```

Comment:
Should we burn an extra byte per scheduler table entry and make this `uint16_t`?

Advantage is that we're less likely to get unresolvable conflicts between the vehicles - we can space things out.